faq/serverless-jobs.mdx

```diff
@@ -92,31 +90,41 @@ Serverless Jobs are billed on a pay-as-you-go basis, strictly on resource consum
 
 **Monthly Cost: €3,31**
 
-## What types of workloads are best suited for Serverless Jobs?
+## Resources and performance
 
-Serverless Jobs is ideal for tasks such as data processing, analysis and various computational operations that can be executed in a non-interactive batch mode. You can execute short and long-running jobs.
+### How do I integrate my serverless solutions with other Scaleway services?
+
+Integration is straightforward. Serverless Functions and Containers can be triggered by events from [Queues](/serverless/messaging/concepts/#queues) and [Topics and Events](/serverless/messaging/concepts/#topics-and-events), and can easily communicate with services like [Managed Databases](/managed-databases/) or [Serverless databases](/serverless/sql-databases/). [Serverless Jobs](/serverless/jobs/) can pull data from [Object Storage](/storage/object), or output processed results into a database. With managed connectors, APIs, and built-in integrations, linking to the broader Scaleway ecosystem is seamless.
 
-## How can I monitor the activity of my Serverless Jobs?
+### Can I update Serverless Jobs resources (vCPU and RAM) at any time?
+
+Yes, resources of your Job Definition can be updated at any time.
+
+Ongoing Job Runs will remain using the resources defined at the time it started.
```

Comment:
```suggestion
Ongoing job runs will keep using the resources defined at the time the update started.
```

Reply:
Not sure about "the update" here, but "it" can refer to the job run, the update, the job, so it could be useful to specify